src/resonanceReconstruction/rmatrix/test/fromENDFResolvedMLBW.test.hpp

```diff
@@ -185,7 +185,7 @@ SCENARIO( "fromENDF - LRF2" ) {
   } // GIVEN
 
   GIVEN( "valid ENDF data for Ag107" ) {
-std::cout << "Ag107" << std::endl;
+
```

Comment:
Not sure why there was a `cout` call previously, but thought I'd point out this a non-map-related change.

Reply:
That's something I forgot to remove when I was testing the MLBW capability. Found it when I compiled all tests for the map changes.